config/swagger.yaml

```diff
@@ -3156,9 +3156,6 @@ definitions:
       tx:
         description: Unsigned transaction object
         $ref: '#/definitions/EncodedHash'
-      tx_hash:
```

Comment:
Could you please re-scan the whole swagger.yaml for `tx_hash`? It looks like it is in there. E.g. 2 lines below it is listed as required field.

Reply:
I will fix